xml/OptionPanel.xml

```diff
@@ -1,108 +1,182 @@
-          <Text class="description">Useful for campaign-play: It resets play areas to allow continuous gameplay in the same savegame.</Text>
-        </VerticalLayout>
-        <Toggle id="showCleanUpHelper" onValueChanged="onClick_toggleOption(showCleanUpHelper)"/>
-      </HorizontalLayout>
+          <!-- Group: fan-made accessories -->
+          <Row class="group-header">
+            <Cell class="group-header">
+              <Panel class="group-header" image="option_header2">
+                <Text class="group-header">FAN-MADE ACCESSORIES</Text>
+              </Panel>
+            </Cell>
+          </Row>
 
-      <!-- Option: show hand helper -->
-      <HorizontalLayout class="group-content">
-        <VerticalLayout class="text-column">
-          <Text class="option-header">Show "Hand Helper"</Text>
-          <Text class="description">Never count your hand cards again! This tool does that for you and can even take "Dream-Enhancing Serum" into account. Also includes a button for randomly discard a card.</Text>
-        </VerticalLayout>
-        <Toggle id="showHandHelper" onValueChanged="onClick_toggleOption(showHandHelper)"/>
-      </HorizontalLayout>
+          <!-- Option: show token arranger -->
+          <Row class="option-text">
+            <Cell class="option-text">
+              <VerticalLayout class="text-column">
+                <Text class="option-header">Show "Token Arranger"</Text>
+                <Text class="description">See the contents of the chaos bag at a glance! This tool displays a sorted table of the tokens to allow easier guessing of your odds.</Text>
+              </VerticalLayout>
+            </Cell>
+            <Cell class="option-button">
+              <Toggle id="showTokenArranger" onValueChanged="onClick_toggleOption(showTokenArranger)"/>
+            </Cell>
+          </Row>
+
+          <!-- Option: show clean up helper -->
+          <Row class="option-text">
+            <Cell class="option-text">
+              <VerticalLayout class="text-column">
+                <Text class="option-header">Show "Clean Up Helper"</Text>
+                <Text class="description">Useful for campaign-play: It resets play areas to allow continuous gameplay in the same savegame.</Text>
+              </VerticalLayout>
+            </Cell>
+            <Cell class="option-button">
+              <Toggle id="showCleanUpHelper" onValueChanged="onClick_toggleOption(showCleanUpHelper)"/>
+            </Cell>
+          </Row>
+
+          <!-- Option: show hand helper -->
+          <Row class="option-text">
+            <Cell class="option-text">
+              <VerticalLayout class="text-column">
+                <Text class="option-header">Show "Hand Helper"</Text>
+                <Text class="description">Never count your hand cards again! This tool does that for you and can even take "Dream-Enhancing Serum" into account. Also includes a button for randomly discard a card.</Text>
```

Comment:
Grammar nit: "can even take"... sounds like it's possible but doesn't necessarily.  "will take" or "will handle Dream-Enhancing Serum" (which is a bit shorter)

Reply:
Reason is that DES still requires the user to tick a box (though this could actually be handled by script - not sure if that would be worth it though)

Reply:
We should change that to be automatic detection :)  But fine for now.